### DISCUSSION TITLE
Issue with Multiple Versions of Video Files

### DISCUSSION THREAD

**Discussion by MercianWhitey** [jellyfin/jellyfin#13128] [Q&A] [answered]
I'm slowly updating my 1080p Blu-ray ripped movie collection to 4K Blu-ray rips but keeping the original 1080p rip for the time being. The issue is Jellyfin doesn't see the two rips as different versions but two separate files of the same movie. I've followed the naming convention in the wiki, this worked recently when I ripped Aliens that had the theatrical version and the extend version on the disc. These two rips appear as "versions" that I can select. The other slight annoyance is that when a movie have been partially played, both versions of the movie (if there's a 4K and 1080p version) appear in the "Continue Watching" section in the main menu.

What am I doing wrong here? Can't Jellyfin add versions if one file has already been previously added to the library?

**Comment by TheMelmacian** [comment, marked as answer]
For this to work properly, the folder and the movie files (without the label part) need to be exactly the same. If the folder name differs from the movie names, jellyfin won't group them automatically.
Adding versions to already existing movies should work if the folder and madia file names are correct.

On rare occasions I encountered a problem after adding a new version, but deleting the nfo files (if nfo saving is enabled) and forcing a rescan solved it every time.

**Reply by MercianWhitey** [reply]
Thanks for the tip, in my case the folder names didn't match the file names. I've been through and changed them and now versions is working!

Thanks again!

**Reply by TheMelmacian** [reply]
Glad to help.
If you mark my comment as the correct answer, the question is marked as resolved. This will help others to find the correct answer, if they have a similar problem, and gives me a little bit of credit.
